buildenv/jenkins/tapVerification/aqaTap.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
```

Comment:
Has this file been shellchecked?

Reply:
Is that a requirement? If so, do we have a doc for instruction?

Reply:
I ran the script at https://www.shellcheck.net/ and updated the script if the suggestion is applicable.